File: IT/1/Funzioni/Esercizi/Java/Esercizio14.java
/* file Esercizio14.java */

public class Esercizio14 {
	static int cube(int n) {
		return n * n * n;
	}

	public static void main(String [] args) {
		for (int n = 1; n <= 10; n++) {
			System.out.println(n + " al cubo = " + cube(n));
		}
	}
}

File: IT/1/Funzioni/Esercizi/C++/esercizio14.cpp
/* file esercizio14.cpp */

#include <iostream>

using std::cout;
using std::endl;

int cube(int n) {
	return n * n * n;
}

int main()
{
	for (int n = 1; n <= 10; n++) {
		cout << n << " al cubo = " << cube(n) << endl;
	}
}

